src/main/resources/fabric.mod.json

```diff
@@ -3,7 +3,7 @@
   "id": "imblocker",
   "version": "${version}",
   "name": "IMBlockerFabric",
-  "description": "IMBlockerFabric is a mod that It can Hide IM(InputMethod) when you don't need to input text! This mod fixes the typing bug exist in Minecraft which annoying Asian players for a long time. You can now type multi-byte characters in game using Input Methods (tested in windows platform, with chinese and japanese)",
+  "description": "IMBlockerFabric is a mod which hides the IME (Input Method Editor) where entering text is not required.",
```

Comment:
so shamed for my wake english

Reply:
No one is perfect in grammar so atleast you got someone else to correct it for you.